lib/lev/status.rb

```diff
@@ -7,28 +7,53 @@ class Status
     STATE_COMPLETED = 'completed'
     STATE_FAILED = 'failed'
     STATE_KILLED = 'killed'
+    STATE_UNKNOWN = 'unknown'
 
     STATES = [
       STATE_QUEUED,
       STATE_WORKING,
       STATE_COMPLETED,
       STATE_FAILED,
-      STATE_KILLED
+      STATE_KILLED,
+      STATE_UNKNOWN
     ].freeze
 
-    attr_reader :uuid
+    def initialize(attrs = {})
+      attrs.each do |k, v|
+        instance_variable_set("@#{k}", v)
+      end
+
+      @uuid ||= SecureRandom.uuid
```

Comment:
Looks like we now sometimes call this thing `uuid` and sometimes `id`.  Any reason to not just switch all the way to making it an `id` attribute?  So the only use of the text `uuid` would be on `SecureRandom.uuid`.